airflow/dag_processing/manager.py

```diff
@@ -574,6 +576,7 @@ def _run_parsing_loop(self):
                     pass
                 elif isinstance(agent_signal, CallbackRequest):
                     self._add_callback_to_queue(agent_signal)
+                    self.log.warning("_add_callback_to_queue; agent signal; %s", agent_signal)
```

Comment:
Why is this a warning? Should it be info instead?

Reply:
I think this can be debug instead. I had added this for testing locally.